monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -107,9 +108,10 @@ class ConfigurePageComponent extends AuthComponent {
         this.setState({
           configuration: monkeyConfig,
           selectedPlugins: {
-              'propagation': new Set(Object.keys(_.get(monkeyConfig, EXPLOITERS_CONFIG_PATH))),
-              'credentials_collectors': new Set(Object.keys(_.get(monkeyConfig, CREDENTIALS_COLLECTORS_CONFIG_PATH)))
-          },
+              [CONFIGURATION_TABS.PROPAGATION]: new Set(Object.keys(_.get(monkeyConfig, EXPLOITERS_CONFIG_PATH))),
```

Comment:
I would place the duplicated code into a function
`new Set(Object.keys(_.get`

Reply:
+1